challenges/leftJoin/__tests__/left-join.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const leftTest = require('../left-join.js');
+
+
+describe('Testing leftjoin', () => {
+  it('Test leftjoin', () => {
+    /*
+        const ht = new hashImport.HashTable(17);
+
+        ht.set('salmon', '#FA8072');
+
+        expect(ht.get('salmon')).toEqual('#FA8072'); */
+  });
+});
+
```

Comment:
Looking great! Once you have some time to come back and add tests Nick, I will regrade for you!